src/AWS.Deploy.DockerEngine/DockerFile.cs

```diff
@@ -63,7 +63,7 @@ public void WriteDockerFile(string projectDirectory, List<string>? projectList)
                     projects += $"COPY [\"{projectList[i]}\", \"{projectList[i].Substring(0, projectList[i].LastIndexOf("/") + 1)}\"]" + (i < projectList.Count - 1 ? Environment.NewLine : "");
                 }
 
-                projectPath = projectList.Where(x => x.EndsWith(_projectName)).FirstOrDefault();
+                projectPath = projectList.First(x => x.EndsWith(_projectName));
```

Comment:
Same comment as above.

Reply:
In this case, the project name will always be in there. If the projectName does not exist, projectList will be null alltogether. We validate this in another function.